### ISSUE TITLE
Questions regarding MCAN fot GQA

### ISSUE THREAD

**Issue by shaadow09** [gqaood/GQA-OOD#1]
Hi Corentin Kervadec , thanks for giving the code and the awesome work.When i run the evaluation code, i got some problems about metrics. After changing the code of MCAN, i got lower accuracy on each metric. When i trained the model , i ued the default
configure. I got  the acuracy  (tai, head, all) : 43.46, 51.99 48.75 
Can you give me more details about your MCAN code for GQA or GQA-OOD ?

**Comment by gqaood** [comment]
Hi, thank you for your interest !

In the paper, I used the publicly available open-vqa code (https://github.com/MILVLG/openvqa) with default config (note that I only train on the training set, not train+val). Then I evaluate on GQA (overall acc: 56.3%) and GQA-OOD (tail:46.5%, head:53.4%, all:50.8%) testdev splits.

May I ask you what score do you obtain on the standard GQA testdev split?

Corentin
